### PR TITLE
docs: added switchboard docs

### PR DESCRIPTION
**Pull request by anoushk1234**

### CODE REVIEW

File: docs/integrations/switchboard.md [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+
+  </template>
+
+  </SolanaCodeGroupItem>
+  
+
+</SolanaCodeGroup>
+
+## How to Create a Feed from the Publisher
+THe official Switchboard documentation has an in-depth walk-through of how to create a feed from the publisher.
```
Comment (gallynaut) [inline docs/integrations/switchboard.md:323]: The*

File: docs/integrations/switchboard.md [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+Check it out [here](https://docs.switchboard.xyz/feed/publisher).
+
+## Other Resources
+
+- [Client Api Docs](https://docs.switchboard.xyz/api/ts/)
```
Comment (gallynaut) [inline docs/integrations/switchboard.md:328]: Add resource: Switchboard V2 SDK - https://github.com/switchboard-xyz/switchboard-v2

File: docs/integrations/switchboard.md [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+    - name: robots
+      content: index,follow,noodp
+  - - meta
+    - name: googlebot
+      content: index,follow
+---
+
+# Switchboard
+
+Switchboard is a protocol that allows builders to unlock the power of Solana by creating high performance data feeds from any API. Switchboard can be used for various applications like getting current price of an asset for lending, getting the current weather or flight data and determining current NBA match status.
```
Comment (gallynaut) [inline docs/integrations/switchboard.md:41]: Switchboard is an Oracle protocol that enables developers to source data on-chain for a variety of use cases such as price feeds, NFT floor prices, sport statistics, or even verifiable randomness. In a general sense, Switchboard is an off-chain resource developers can invoke to bridge high integrity data on-chain and power the next generation of web3 and DeFi.

**Comment by gallynaut** [comment]
Can we modify the page structure so it looks like this:

<hr />

```
## Data Feeds
### How to Read Feed in Program
### How to Read Feed on Client Side
### How to Create Feed on Client Side
### How to Create Feed from Publisher

## Verifiable Randomness
### How to Read VRF Account in Program
### How to Read VRF Account on Client Side
### How to Create VRF Account on Client Side
### How to Request Randomness on Client Side
### How to Request Randomness in Program

## Localnet Integration
Switchboard allows you to run an oracle on localnet so you can simulate your on-chain workflow. Check out [Switchboard’s developer docs](https://docs.switchboard.xyz/developers/localnet) for more information.

## Resources
### APIs and Libraries
 - [Switchboard Task Types](https://docs.switchboard.xyz/api/tasks)
 - [Rust API Docs](https://docs.rs/switchboard-v2/latest/switchboard_v2/)
 - [Typescript API Docs](https://docs.switchboard.xyz/api/ts)
 - [Python API Docs](https://docs.switchboard.xyz/api/py)
 - [CLI Docs](https://docs.switchboard.xyz/api/cli)
### Examples
 - [[Client] Custom Data Feed Walkthrough](https://github.com/switchboard-xyz/switchboard-v2/tree/main/packages/feed-walkthrough)
 - [[Program] Anchor Feed Parser](https://github.com/switchboard-xyz/switchboard-v2/tree/main/programs/anchor-feed-parser)
 - [[Program] Anchor VRF Parser](https://github.com/switchboard-xyz/switchboard-v2/tree/main/programs/anchor-vrf-parser)
### More Information
 - [Protocol Documentation](https://docs.switchboard.xyz/introduction)
 - [SuperteamDAO Deep Dive](https://crawling-cent-d6b.notion.site/The-Switchboard-Deep-Dive-717df6ba0b92465e8118351466257a0f)
```

**Review by gallynaut — APPROVED** [review]
looks good to me. thanks!

**Comment by anoushk1234** [comment]
hey @jacobcreech when will this get merged?

**Review by jacobcreech — APPROVED** [review]

**Pull request merged**